attributes/package_repo.rb

```diff
@@ -11,8 +11,8 @@
   "default" => node["platform_version"].to_i
 )
 
-default["percona"]["apt"]["key"] = "0x1C4CBDCDCD2EFD2A"
-default["percona"]["apt"]["keyserver"] = "hkp://keys.gnupg.net:80"
+default["percona"]["apt"]["key"] = "1C4CBDCDCD2EFD2A"
+default["percona"]["apt"]["keyserver"] = "hkp://keys.gnupg.net"
```

Comment:
The ':80' was added here to deal with nodes with more restricted outbound traffic policy.  `hkp` uses port 11371 by default